src/lib.rs

```diff
@@ -121,3 +135,10 @@ pub mod collapse;
 ///
 ///   [crate-level documentation]: ../index.html
 pub mod flamegraph;
+
+/// Tool for creating an output required to generate differential flame graphs.
+///
+/// See the [crate-level documentation] for details.
+///
+///   [crate-level documentation]: ../index.html
+pub mod diff_folded;
```

Comment:
```suggestion
pub mod differential;
```

Reply:
Do you also want to change the name of the binary?

Reply:
Hmm, that's a good question... No, I think let's leave that the way it is.